### DISCUSSION TITLE
Ivre without Mongodb

### DISCUSSION THREAD

**Discussion by anoduck** [cea-sec/ivre#1631] [Q&A] [answered]
I hadn't used ivre in several years, and went through the trouble of manually installing it with new pypa/build standard and completed setting up everything. I even took the time to write a shell script, so other's can install it and not go through the difficulties I did. [Shell script shared here](https://gist.github.com/anoduck/56b550bb5a22f73df046aafa5886e292) All of this took time, and effort. 

I admit, the system I have to run ivre on is several years old. So, as you can probably figure out, mongodb's processor requirement and tendency to be resource hungry is the reason I don't use mongodb for anything. I remember the first release that imposed this new requirement, which fortunately I have not needed to run until now. 

Is there not an alternative available to mongodb?

**Comment by p-l-** [comment, marked as answer]
As explained [here](https://github.com/ivre/ivre/issues/1632#issuecomment-2302307534) and [in the documentation](https://doc.ivre.rocks/en/latest/install/installation.html#databases), MongoDB is the only supported implementation. TinyDB is also supported but won't work for large datasets.

But IVRE supports old versions of MongoDB: for now, 3.6 is the oldest supported version; it has been released almost 7 years ago.
